src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int INITIAL_SIZE = 16;
+    private static final int INCREASE_ARRAY_SIZE_INDEX = 2;
+
+    private int size;
+    private int threshold;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = new Node[INITIAL_SIZE];
+        this.size = 0;
```

Comment:
we shouldn't assign 0 on size field, because int had 0 by default 
```suggestion
```

Reply:
agree, done